### DISCUSSION TITLE
База Данных PD

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#600] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/доктор_хаггитс/32/2398_2.png) [Доктор_Хаггитс](https://forum.octothorp.team/u/Доктор_Хаггитс) ([#10314](https://forum.octothorp.team/p/10314))

**БАЗА ДАННЫХ ПОЛИЦЕЙСКОГО ДЕПАРТАМЕНТА**

Я бы хотел предложить сделать базу данных для полицейского департамента, например что бы подать транспорт в состояние “угон” нужно например: [Транспортные Средства&gt;Подать в розыск&gt; Номера Авто, Цвет и так далее. По типу как на проекте yrp.life. Потому что полиции реально не хватает такой базы данных, так как эти крики, кто кого перекричит на рации - всем уже надоело. И постоянно спрашивать повторите о машине которая в угоне. Я бы сделал окно в “Контекстном Меню” например: нажимаешь С и потом База Данных, и открывается окошко где можно написать разыскиваемую машину или-же человека.

**Comment by msmr-dev** [comment]


**Reply by chelog** [reply]
Нажми на стрелочку под первым сообщением

**Comment by 1BillBlack** [comment]
Поддерживаю. Очень необходимо.
Все в эту базу.
Можно через "С-Департаменты-База данных". Там делаем базу розыска, тикетов отдельно (потому что надоело, что поданный в розыск по тикетам отображается по всем камерам и у офицера каждую минуту сигналит, что он проехал мимо камеры). Также сделать, чтобы по прошествии 72-х часов человек автоматически подавался в розыск за неуплату, туда же базу данных, чтобы подавать в розыск, куда можно добавлять заметки другим офицерам (чтобы те знали о человеке информацию).
Снизу - список, что хорошо бы добавить в эту базу.
![База](https://user-images.githubusercontent.com/91433643/134821990-09c31f66-c8ab-4f9e-b1a1-2797f3ad4d6a.png)

**Comment by LENFYS** [comment]
Категорически за!
